layers/meta-resin-qemu/recipes-core/images/resin-image.bbappend

```diff
@@ -0,0 +1,5 @@
+IMAGE_FSTYPES_append_qemuall = " resin-sdcard"
+
+# Customize resin-sdcard
+RESIN_IMAGE_BOOTLOADER_qemuall = ""
+RESIN_BOOT_PARTITION_FILES_qemuall = ""
```

Comment:
Should be "qemux86-64" instead of qemuall. As the PR states it so far, only "qemux86-64" is supported so far